wurst/objects/units/TrollUnitTextConstant.wurst

```diff
@@ -97,7 +97,7 @@ public constant  NORMAL_SPELLS_PRIEST       = commaList(ABILITY_THE_GLOW)
 public constant  NORMAL_SPELLS_BEAST_MASTER = commaList(ABILITY_PET_SPELLS, ABILITY_SPIRIT_BEAST )
 public constant  NORMAL_SPELLS_THIEF        = commaList(ABILITY_TELEPORT)
 public constant  NORMAL_SPELLS_SCOUT        = commaList(ABILITY_PING_ENEMY)
-public constant  NORMAL_SPELLS_GATHERER     = commaList(ABILITY_SALVE_RECIPE)
+public constant  NORMAL_SPELLS_GATHERER     = commaList(ABILITY_TELE_GATHERER)
```

Comment:
Really dumb question, but does Gatherer not have the ability to ping all items anymore? Also does he not have room for 2 normal abilities? His second row could be Salves, Radars, Telegather, Skill Menu I think.

Reply:
Yeah the ping ability is now leveled remember, it also reveals the item on the groud. I also though it could have a 10% to telegather back to fire with telegathering enabled (or something similar). Also every sub class have 2 available slots for ability (one of the base ability and the other for leveling one). I dont think we have space for salve recipe but we can add a level to the spellbook (with the recipe in it) and let gatherer have it by default. (Should still be able to get level 2 3 4 spellbook by leveling)